### ISSUE TITLE
[WCAG2.1: Select a pilot project for implementing new success criteria

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#20811]
## Issue Description
Once we have the support documentation, training, and testing items in place, we need to select a VFS team for a pilot program. We want to work with this team to build their product to WCAG2.1 from the beginning of the software development lifecycle.

## Tasks
- [ ] Engage the VSP Service Design team for messaging the ask to VFS teams
- [ ] Raise awareness through Slack channels
- [ ] Meet with PMs and stakeholders to identify candidate projects
- [ ] Select the best candidate from a pool of 2-3 potentials

## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [x] **Attached to an Epic** (what body of work is this a part of?)
- [x] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `tools-be`, `tools-fe`)
- [x] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [x] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)
